internal/models/versionanalyzers/helm_version_analyzer.go

```diff
@@ -57,6 +58,9 @@ type helmChart struct {
 
 func (a *HelmVersionAnalyzer) ReadVersion(projectRootDir string, extraArgs *models.ArgumentValues) (string, error) {
 	rootDir := filepath.Join(projectRootDir, *(*extraArgs)[HelmArgumentKeyRootDir])
+	if _, err := os.Stat(filepath.Join(rootDir, "Chart.yaml")); errors.Is(err, os.ErrNotExist) {
+		return "0.0.0", nil
```

Comment:
Please also `log.info` here.

Reply:
I put log in `CompareVersions` function.